demo-project/main.ts

```diff
@@ -3,8 +3,21 @@ import express from 'express'
 import cors from 'cors'
 import cookieParser from 'cookie-parser'
 import { expressjwt as jwt } from 'express-jwt'
+import bodyParser from 'body-parser'
+import * as jsonwebtoken from 'jsonwebtoken';
+import qs from 'qs'
 
 const app = express()
+app.use(bodyParser.json())
+
+declare global {
```

Comment:
Can you add a comment explaining why this is needed?

Reply:
And can this be moved to a different file so that it does not clutter the main file?

Reply:
Yes, I think a ts-ignore is definitely better than... this....

Reply:
ced32e6